lib/internal/initialParams.js

```diff
@@ -1,8 +1,9 @@
-import rest from './rest';
+import slice from './slice';
 
 export default function (fn) {
-    return rest(function (args/*..., callback*/) {
+    return function (/*...args, callback*/) {
+        var args = slice(arguments);
         var callback = args.pop();
```

Comment:
I'm gonna leave as-is.  The streamlined version of slice doesn't support the 3rd `end` param.